tests/nnapi/specs/V1_2/pow_dynamic_nnfw.mod.py

```diff
@@ -0,0 +1,31 @@
+import dynamic_tensor
+
+model = Model()
+
+model_input1_shape = [2, 3]   # first input shape of Sub. 12 float32s
```

Comment:
` # first input shape of Sub. 12 float32s`
sub -> pow
Or you may want to remove this comment because I found that people make too many mistake with copy&paste of comment...

Reply:
`you may want to remove this comment because I found that people make too many mistake with copy&paste of comment...`

I agree..
I will remove that